src/components/Nav/Nav.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Nav.css';
+import { logout } from '../../services/users';
+
+export default function Nav({ currentUser, setCurrentUser }) {
+  const handleLogout = async () => {
```

Comment:
logout works well, I was able to test several times and it prompted me to log in. good work.